src/main/java/com/crud/tasks/controller/TrelloController.java

```diff
@@ -6,18 +6,37 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
 import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.constraints.NotBlank;
+import java.util.ArrayList;
 import java.util.List;
+import java.util.Optional;
 
 @RestController
 @RequestMapping("/v1/trello")
 public class TrelloController {
     @Autowired
+    @NotBlank
```

Comment:
I am not sure if NotBlank annotation is required, @Autowired has option required, and by default it is set to true.

Reply:
Sorry! They're debris after trials. I'll remove it in next commit.